Adds Diversity, Equality, and Inclusion (DEI) mission statement

exoticDFT opened the pull request:
<!--- Thank you for opening a pull request! Here are some helpful tips:
     
      1. To solicit reviewers: 
           the @usrse-maintainers are automatically notified when you open this pull request
           If you need additional reviewers you can:
               (if you have permission to do so) assign the label "reviewers-needed" 
               if you are on the usrse slack, post a link to your PR there and ask for reviewers

      2. To get help:
           you can ask the question directly in this pull request for @usrse-maintainers
           you can ask a question in the #website channel of usrse.slack.com
           for important issues, you can @usrse-admin to alert all admins of the repository (use sparingly)
 -->

<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail. -->
Simple addition of a new page to host our DEI Mission Statement and includes a reference to DEI on our main Mission page.

- Adds new page for our full DEI mission statement.
- Adds a shorter statement about DEI to our general Mission Page and links to our full DEI statement.

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->
The DEI working group has been working towards creating a statement about diversity, equality, and inclusion. This PR addresses that work by posting our statement to the US-RSE website.

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] I have posted the link for the PR in the usrse slack (#website) to ask for reviewers
- [x] I have previewed changes locally
- [x] I have updated the README.md if necessary

cc @usrse-maintainers

vsoch commented on pages/community/dei-mission-statement.md:
```diff
@@ -0,0 +1,38 @@
+layout: page
+title: DEI Mission Statement
+subtitle: Diversity, Equality, and Inclusion at US-RSE
+bigimg:
```
```suggestion
```

exoticDFT replied:
This issue also exists in the "mission.md" file. I'm going to change it there as well.

vsoch commented on pages/community/dei-mission-statement.md:
```diff
@@ -0,0 +1,38 @@
+will continue to be an integral part of our community.
+
+The US-RSE Association is committed to providing an [inclusive environment with
+equitable treatment]({{site.url}}/code-of-conduct) for all and to promoting and
```
```suggestion
equitable treatment]({{site.url}}/code-of-conduct/) for all and to promoting and
```

exoticDFT replied:
You know I originally had this, but then looked at some other pages and removed it to follow the others. I'll add the `/` to the ends of the links.

vsoch commented on pages/community/dei-mission-statement.md:
```diff
@@ -0,0 +1,38 @@
+national origin, and ethnicity.
+ 
+The US-RSE Association will:
+- Continually educate ourself through research on best practices focused on
```
This sounds more appropriate because there are many of us.
```suggestion
- Continually educate ourslves through research on best practices focused on
```

danielskatz replied:
```suggestion
- Continually educate ourselves through research on best practices focused on
```

exoticDFT replied:
If I recall correctly, we specifically had "ourselves" previously and it was commented in the Google Doc to be replaced with "ourself". Although maybe that was prior to some other change. I agree "ourselves" sounds more grammatically correct.

vsoch commented on pages/community/mission.md:
```diff
@@ -34,3 +34,12 @@ to target our activities and actions to serving these three aspects:
+  development of a DEI program for our organization. DEI discussions are and
+  will continue to be an integral part of our community.
+
+  Please see our [full statement.]({{site.url}}/dei-mission)
```
```suggestion
  Please see our [full statement.]({{site.url}}/dei-mission/)
```

exoticDFT replied:
@vsoch Oh actually my comment above was after looking at this file. As earlier in this page we have the "inclusion definition" link without the `/`. So just to double check, I should go ahead and update that link as well to match the `/` suggestion here?

vsoch replied:
Yeah generally the structure of our site has permalinks with a trailing slash: https://github.com/USRSE/usrse.github.io/blob/1569f845ce4ffbdcc873a49eae369ccffd67ca2e/_config.yml#L180

vsoch commented:
@exoticDFT you can ignore the URL check failure - looks like Zenodo is temporarily down:
![image](https://user-images.githubusercontent.com/814322/114614020-3e8b2800-9c61-11eb-94a6-9e42eac9ff49.png)

exoticDFT commented:
With the assumption that the recent commit addresses the issues, is https://1193-120906408-gh.circle-artifacts.com/0/usrse.github.io/dei-mission/index.html the link we'd want to include in the draft email Lance wrote?

vsoch commented:
Include what?

exoticDFT commented:
> Include what?

Uh nevermind. I was thinking for some reason we'd have a different process for reviewing the **content** of the statement. Rather than linking to this PR as all whole. Originally I was thinking that link I had in the above comment would be what people would look at and then make comments in Slack or something.

vsoch commented:
Oh yeah, I'm not sure! I think (from what I read) the entire PR was going to be linked for comments.

exoticDFT commented:
> Oh yeah, I'm not sure! I think (from what I read) the entire PR was going to be linked for comments.

I totally missed your comment about this specific review process in the email draft. My original concern was exactly the same, which is probably why I thought we would do a different process for the content review. I'm also not sure how comfortable people would be using the GitHub PR process for this type of review.

vsoch commented:
@exoticDFT totally agree - I'm worried PRs are a tangle of comments (and also limited to people with GitHub accounts.)

mtbc commented on pages/community/mission.md:
```diff
@@ -34,3 +30,13 @@ to target our activities and actions to serving these three aspects:
+## 4. Diversity, Equality, and Inclusion
+
+  We believe Diversity, Equity, and Inclusion (DEI) are core values of the
+  US-RSE Association. A dedicated team of active members continue to lead the
```
```suggestion
  US-RSE Association. A dedicated team of active members continues to lead the
```
My impression is that the US tends toward a singular "team" and the UK toward a plural "team" but happy to be corrected.

vsoch replied:
@mtbc I think we are going to edit first in the Google Doc and come back to here? We should check with @exoticDFT and @lparsons

exoticDFT commented:
This PR is now updated with the finalized version of the DEI Mission Statement. I verified it matched the document, after copying and pasting. But if others want to double check as well that would be great! Otherwise, I think this should be good to merge

vsoch commented on pages/community/dei-mission-statement.md:
```diff
@@ -0,0 +1,35 @@
+nurturing, and robustly inclusive community. We believe that the amplification
+of diverse perspectives is essential for driving innovation, promoting
+creativity, and encouraging engagement for the success of RSEs. We welcome and
+respect individuals of all dimensions of diversity, including but not limited
```
I didn't come to the final meeting, and maybe it's okay because it says "including but not limited to" but we don't make any reference to any kind of neurodiversity. At least that one feels important to me.

exoticDFT replied:
This was one of the points we spent a bit more time on. We had a few additional "dimensions" from the "public" comment period. At the time we decided to cut a couple as we didn't believe a long running list or potentially ever changing list was appropriate. This is why we decided to add the "including but not limited to" and not include everything.

I'm not sure that's the best answer, but from my recollection that was the reasoning. What do others think and feel about this specific sentence?

vsoch replied:
I think if we are hitting the major categories, we should include something (also general) like neurocognitive differences. https://trainingindustry.com/articles/diversity-equity-and-inclusion/neurodiversity-in-the-age-of-di-how-do-training-professionals-make-an-impact/. That covers a whole lot of stuffs.

danielskatz replied:
I'm not sure - on the one hand, we can't include everything, but on the other, we do want to include the major groups. I'm honestly not sure which category this fits it

vsoch commented on pages/community/mission.md:
```diff
@@ -34,3 +30,15 @@ to target our activities and actions to serving these three aspects:
+  Equity, and Inclusion (DEI) activities for our organization, led by a
+  dedicated team of active community members.
+
+  Please see our full [DEI mission statement.]({{site.url}}/dei-mission/)
```
Just thinking ahead - this URL for the new site will need to be under a dei namespace. Either something like `/working-group/dei/` or just give working groups a base url like `/dei/`. So this will work for now,but just forward thinking it will need to be made into a redirect.

lparsons commented on pages/community/dei-mission-statement.md:
```diff
@@ -0,0 +1,35 @@
+of diverse perspectives is essential for driving innovation, promoting
+creativity, and encouraging engagement for the success of RSEs. We welcome and
+respect individuals of all dimensions of diversity, including but not limited
+to race, color, caste, economic status, gender expression, gender identity,
+sexual orientation, disability, age, religion (or lack thereof), national
+origin, and ethnicity.
```
I think that's a good addition of a general category that we missed. @exoticDFT could you make that change? 
```suggestion
to race, color, caste, economic status, gender expression, gender identity,
sexual orientation, disability, neurocognitive differences, age, 
religion (or lack thereof), national origin, and ethnicity.
```

exoticDFT replied:
Sounds great! Will do.

lparsons reviewed (commented):
Excellent job, and thanks to all for the suggestions.

With the addition of "neurocognitive differences" I think we'll be ready to ask for formal Steering Committee approval for the update to the mission statement. Once that change is made, I'll have the committee take a final look.

vsoch reviewed (approved):
This looks great! I'm good to merge if others agree.

vsoch commented:
And good call on the "lack thereof" for religion.

danielskatz commented:
We don't want to merge this yet.  Because it's a high-level change for the US-RSE organization, we want the US-RSE Steering Committee to formally approve it as well, and @cosden will lead that activity now that we have a final version.

vsoch commented:
Take it away, @cosden !

cferenba commented on pages/community/mission.md:
```diff
@@ -2,21 +2,17 @@
 
-
 The US-RSE Association is centered around three main goals.  We aim
 to target our activities and actions to serving these three aspects:
```
Since we've added a fourth bullet below, the two instances of "three" here need to change to "four."

exoticDFT replied:
@cferenba Great catch! Updated with the "four" and removed the second reference, since it was redundant. "these aspects" should be adequate.

christophernhill reviewed (commented):
Hi @vsoch , did you mean to include this commit ( https://github.com/USRSE/usrse.github.io/pull/436/commits/bfb673416bfd617a41e675893ae5c557da9d25c4 ) in https://github.com/USRSE/usrse.github.io/pull/436 ?

vsoch commented:
Yes the CI was not passing.

cosden commented:
I'm happy to report the changes and additions to the US-RSE mission contained in this PR were unanimously approved by the steering committee today July 14, 2021. 

My approval of this PR is on behalf of the entire steering committee. 

Thanks to everyone involved to help put this together!

vsoch commented:
Woot! Awesome work everyone!
